### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by KaustubhM1997**
Increased Grid component's size
Added font awesome cdn link to installation
Made toast/snackbar functional

### CODE REVIEW

File: grids/grids.css [hunk 1]
```diff
@@ -13,18 +13,27 @@
 
 .grid-two-container {
   grid-template-columns: repeat(2, 1fr);
+  height: 10vh;
+  /* align-self: center; */
```
Comment (horsemaker) [inline grids/grids.css:17]: You could remove this comment if not needed.

File: grids/grids.css [hunk 2]
```diff
@@ -13,18 +13,27 @@
...
+  
 }
 
 .grid-three-container {
   grid-template-columns: repeat(3, 1fr);
+  height: 10vh;
 }
 
 .grid-two-container > *,
 .grid-three-container > * {
```
Comment (horsemaker) [inline grids/grids.css:27]: nice use of `>` & `*`

File: grids/grids.css [hunk 3]
```diff
@@ -33,3 +42,11 @@
     grid-template-columns: 1fr;
   }
 }
+
+@media only screen and (max-width: 972px){
```
Comment (horsemaker) [inline grids/grids.css:46]: I see you have media queries for `970px` as well as `972px`. They are pretty close values. Could we merge them into one?

File: toast/toast.css [hunk 4]
```diff
@@ -25,17 +26,20 @@
...
+ 
+
 }
 
 .toast-msg {
   margin-right: 2rem;
 }
 
-.toast-btn {
+#toast-btn {
```
Comment (horsemaker) [inline toast/toast.css:42]: You should avoid using id selector.

File: toast/toast.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+const toastClose = document.querySelector("#toast-btn");
+
+const closeContainer = () => {
+
+    console.log("hello");
+ toastContainer.classList.add("hidden");
+
+ console.log(toastContainer);
+ console.log(toastClose);
```
Comment (horsemaker) [inline toast/toast.js:11]: You could remove the consoles in this file.

**Review by horsemaker — APPROVED** [review]

**Review by vijaytembugade — APPROVED** [review]
Most of the things are pointed out by yash. 
Good work

**Pull request merged**